ext/ftp/tests/ftp_site_basic.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+ftp_site function basic functionality
+--CREDITS--
+Gabriel Caruso (carusogabriel34@gmail.com)
+--SKIPIF--
+<?php require 'skipif.inc'; ?>
+--FILE--
+<?php
+require 'server.inc';
+
+$ftp = ftp_connect('127.0.0.1', $port);
+$ftp or die("Couldn't connect to the server");
+
+ftp_login($ftp, 'user', 'pass') or die("Couldn't login into the server");
+
+var_dump(ftp_site($ftp, 'CHMOD 0600 file'));
+var_dump(ftp_site($ftp, 'foo bar baz'));
+?>
+--EXPECTF--
+bool(true)
+string(18) "SITE foo bar baz
```

Comment:
If `ftp_site` returns `bool`, return this `string` is a bug?

Reply:
https://github.com/php/php-src/blob/master/ext/ftp/tests/server.inc#L529

If you want to test invalid input, it also needs to be handled in the FTP server implementation, otherwise it's going to dump the request.

Reply:
@nikic Thanks for guide me :blush:

Reply:
I removed the `dump_and_exit`. Was that the correct solution?

Reply:
I think the dump_and_exit should remain, but a new case for the specific invalid input you are provided can be added to the if/else list of inputs.

Reply:
I guess I handle it :sweat_smile: